.env.sample

```diff
@@ -33,3 +33,6 @@ VAPID_PRIVATE_KEY=
 VAPID_SUBJECT_MAILTO=
 
 APPLICATION_URL="http://localhost:3000"
+
+DATABASE_URL=postgres://postgres:postgres@conference-app-db:5432
+REDIS_URL=redis://conference-app-redis:6379
```

Comment:
These environment variables are very important for rails application. Could you move it to top of the `.env.sample` ?

Reply:
Thank you for the suggestion!
I actually thought that these values should only be used when developing locally with docker compose, so I have split the env file with 997e625